packages/server/package.json

```
@@ -20,7 +20,7 @@
     "@types/fetch-mock": "^7.3.2",
     "@types/jest": "^24.0.25",
     "@types/lodash.debounce": "^4.0.6",
-    "@types/node": "^14.14.25",
+    "@types/node": "^18.14.6",
```

There was some talk in Client Side Infra recently suggesting that for `@types` packages we should be specifying `~` range, as this is the only way to guarantee compatibility according to Definitely Typed https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library.

```suggestion
    "@types/node": "~18.14.6",
```

---

oh right, presumably that applies to all `@types` dependencies in this project?

---

Yep I think that's right (I just picked this one as it changed in this PR)

---

ok, it's a good call but I think I'd like to keep that change separate from this PR

---

That sounds fine (definitely agree that other types packages should change in another PR, could be convinced either way about `@types/node`!)